src/librbd/api/DiffIterate.cc

```diff
@@ -277,7 +277,7 @@ int DiffIterate<I>::execute() {
         ldout(cct, 20) << "object " << object << dendl;
 
         const uint64_t object_no = extents.front().objectno;
-        uint8_t diff_state = object_diff_state[object_no];
+        const uint8_t diff_state = object_diff_state[object_no];
```

Comment:
What is the idea behind constifying this? It does not look like related to the issue?

Reply:
Yes, its not related but in some test diff_state had garbage value.